### DISCUSSION TITLE
What do I change?

### DISCUSSION THREAD

**Discussion by Braxn** [simulationcraft/simc#6195] [Q&A] [answered]
Try to import and get this :

# Profile will be downloaded into a new tab.
#
# Clicking Simulate will create a simc_gui.simc profile for review.

# Unable to generate profile from: 

# Actor 'Braxnbjørn': Item 'cowl_of_haunting_precognition' Slot 'head': Error decoding gems from '': No gem data for id 187310.

**Comment by vituscze** [comment, marked as answer]
It seems you're using an old version of simc, from before 9.1 at least. Get the latest version and try again.

**Reply by Braxn** [reply]
Got it thank you
